velox/docs/monitoring/stats.rst

```diff
@@ -69,3 +69,41 @@ These stats are reported only by HashBuild and HashAggregation operators.
      - nanos
      - Time spent on building the hash table from rows collected by all the
        hash build operators. This stat is only reported by the HashBuild operator.
+
+Spilling
+--------
+These stats are reported by operators that support spilling.
+
+.. list-table::
+   :widths: 50 25 50
+   :header-rows: 1
+
+   * - Stats
+     - Unit
+     - Description
+   * - op.spillFillTime
```

Comment:
What is the motivation for adding "op." prefix?

Reply:
operator-specific stats, follow the convention of hashtable prefix above?

Reply:
Let's remove this prefix.

If you want to match "hashtable" prefix, then you need to use "spill" prefix. Notice that "hashtable.xxx" stats are also operator-specific, but they do not have "op." prefix.

Reply:
Got it, removed.